static/css/style.css

```diff
@@ -1,5 +1,7 @@
 /* HackRU Fall 2017 Custom Stylesheet */
```

Comment:
remove this damn comment. and all others of this type. How silly of us.

Reply:
ohh, yeah this is much needed.